xformers/components/attention/csrc/cuda/mem_eff_attention/kernel_backward.h

```diff
@@ -400,100 +402,6 @@ struct AttentionBackwardKernel {
     };
   };
 
-  // OLD VERSION - a3f257389709
-  template <int kElementsPerAccess>
-  static __device__ void _computeDi(
```

Comment:
nit: why not taking this function and use it in a single kernel launch instead of writing the (slower) pytorch unfused ops?